pyproject.toml

```diff
@@ -66,6 +62,23 @@ torch = [
     "torchvision>=0.13.0",
     "onnx>=1.12.0,<3.0.0",
 ]
+tf-extras = [
```

Comment:
@frgfm Maybe another idea in mind i really really don't like all these dups :sweat_smile: (Yeah i know it's handled by dependabot but yeah ^^)

Reply:
Something like
```
tf-extras = [
     **python-doctr["tf"] 
     **python-doctr["extras"]
]
```
would be awesome from a deps dict

Reply:
Yup, we can't afford to have all of those. Otherwise it complexifies the installation for users and I bet we'll get more GH issue about resulting unexpected behaviours :/

Reply:
But why don't we simply do a separate extra just for weasyprint, mpl & mplcursors? because we can install multiple extras at once, and we can easily adjust installation instructions

Reply:
> But why don't we simply do a separate extra just for weasyprint, mpl & mplcursors? because we can install multiple extras at once, and we can easily adjust installation instructions

:+1: